sendgrid.env

```diff
@@ -0,0 +1 @@
+export SENDGRID_API_KEY='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'
```

Comment:
I see this file added to the list of files/folders you intend to ignore in .gitignore. Why then is it added to your branch?

Reply:
I forgot to add it to git ignore initially, I'll fix this